pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go

```diff
@@ -271,7 +271,7 @@ type AlertRuleExport struct {
 	ForString            *string                              `json:"-" yaml:"-" hcl:"for"`
 	Annotations          *map[string]string                   `json:"annotations,omitempty" yaml:"annotations,omitempty" hcl:"annotations"`
 	Labels               *map[string]string                   `json:"labels,omitempty" yaml:"labels,omitempty" hcl:"labels"`
-	IsPaused             bool                                 `json:"isPaused" yaml:"isPaused" hcl:"is_paused"`
+	IsPaused             bool                                 `json:"isPaused,omitempty" yaml:"isPaused,omitempty" hcl:"is_paused"`
```

Comment:
The OpenAPI Spec has this as not required (= the generated go client has it as `omitempty`) so I'm reflecting the change here